source/declarations.tex

```diff
@@ -1773,7 +1773,7 @@
 The type of a \grammarterm{parameter-declaration} of a
 function declaration\iref{dcl.fct},
 \grammarterm{lambda-expression}\iref{expr.prim.lambda}, or
-\grammarterm{template-parameter}\iref{temp.param}
+non-type \grammarterm{template-parameter}\iref{temp.param}
```

Comment:
Since this directly refers to a first-level decomposition of the grammar non-terminal _template-parameter_, I think it's actually clearer without "non-type" (which suggests a prior semantic interpretation instead of a syntax-based grammar match).

Reply:
ok, I was also considering this part as a minor point in Issue #7453,
even because parameter-declaration is not possible in case of a type-parameter.

So, I fully trust your opinion to reject it.